org.lflang/src/org/lflang/cli/Lff.java

```diff
@@ -27,219 +30,173 @@
  * @author Marten Lohstroh
  * @author Christian Menard
  * @author Billy Bao
+ * @author Atharva Patil
  */
+@Command(
+    name = "lff",
+    // Enable usageHelp (--help) and versionHelp (--version) options.
+    mixinStandardHelpOptions = true,
+    // TODO: Import version from StringsBundle.properties. 
+    version = "lff 0.3.1-SNAPSHOT")
```

Comment:
This should use `LocalStrings.VERSION` to retrieve the current version.

Reply:
Done!